cg/meta/encryption/encryption.py

```diff
@@ -92,6 +130,127 @@ def get_symmetric_decryption_command(
         decryption_parameters.extend(output_parameter)
         return decryption_parameters
 
+    def create_pending_file(self, pending_path: Path) -> None:
+        """Create a pending flag file."""
+        LOG.info(f"Creating pending flag {pending_path}")
+        if self.dry_run:
+            return
+        pending_path.touch(exist_ok=False)
+
+
+class FlowCellEncryptionAPI(EncryptionAPI):
+    """Encryption functionality for flow cells."""
+
+    def __init__(
+        self,
+        config: dict,
+        binary_path: str,
+        dry_run: bool = False,
+    ):
+        super().__init__(binary_path=binary_path, dry_run=dry_run)
+        self.tar_api = TarAPI(binary_path=config["tar"]["binary_path"], dry_run=dry_run)
+        self.pigz_binary_path: str = config["pigz"]["binary_path"]
+        self.slurm_api: SlurmAPI = SlurmAPI()
```

Comment:
Same here, maybe this api should be passed in the constructor

Reply:
Fixed